public/js/selfoss-events-entriestoolbar.js

```diff
@@ -95,76 +98,81 @@ selfoss.events.entriesToolbar = function(parent) {
+
             return false;
         });
-        
+
         // read/unread
         parent.find('.entry-unread').unbind('click').click(function() {
             var entry = $(this).parents('.entry');
             var id = entry.attr('data-entry-id');
-            var unread = $(this).hasClass('active')==true;
+            var unread = $(this).hasClass('active') == true;
 
             selfoss.ui.entryMark(id, !unread);
-            
+
             // update statistics in main menue and the currently active tag
             var updateStats = function(unread) {
                 // update all unread counter
                 var unreadstats = parseInt($('.nav-filter-unread span').html());
-                if(unread) {
+                if (unread) {
                     unreadstats--;
                 } else {
                     unreadstats++;
                 }
                 selfoss.refreshUnread(unreadstats);
-                    
-                if( selfoss.sourceNavLoaded ) {
+
+                if (selfoss.sourceNavLoaded) {
                     // update unread count on sources
                     var sourceId = entry.attr('data-entry-source');
-                    var sourceNav = $('#source'+sourceId+' .unread');
+                    var sourceNav = $('#source' + sourceId + ' .unread');
                     var sourceCount = parseInt(sourceNav.html());
-                    if(typeof sourceCount != "number" || isNaN(sourceCount)==true)
+                    if (typeof sourceCount != 'number' || isNaN(sourceCount) == true) {
                         sourceCount = 0;
-                    sourceCount = unread ? sourceCount-1 : sourceCount+1;
-                    if(sourceCount<=0) {
-                        sourceCount = "";
-                        $('#source'+sourceId+'').removeClass('unread');
+                    }
+                    sourceCount = unread ? sourceCount - 1 : sourceCount + 1;
+                    if (sourceCount <= 0) {
+                        sourceCount = '';
+                        $('#source' + sourceId + '').removeClass('unread');
                     } else {
-                        $('#source'+sourceId+'').addClass('unread');
+                        $('#source' + sourceId + '').addClass('unread');
                     }
                     sourceNav.html(sourceCount);
                 }
-                
+
                 // update unread on tags
-                $('#entry'+id+' .entry-tags-tag').each( function(index) {
+                $('#entry' + id + ' .entry-tags-tag').each(function() {
                     var tag = $(this).html();
-                    
-                    var tagsCountEl = $('#nav-tags > li > span.tag').filter(function(i){
-                        return $(this).html()==tag; }
+
+                    var tagsCountEl = $('#nav-tags > li > span.tag').filter(function(){
+                        return $(this).html() == tag;
+                    }
                     ).next();
```

Comment:
This should be on the same line as the closing bracket, though I could not find an eslint rule.